frontend_lib/src/tinymceAutoCompleteHelper.js

```diff
@@ -93,36 +94,35 @@ export const tinymceAutoCompleteHandleKeyDown = (event, setState, isAutoComplete
 }
 
 export const tinymceAutoCompleteHandleClickItem = (item, setState) => {
-  tinymce.activeEditor.focus()
-  const selection = tinymce.activeEditor.dom.select('span#autocomplete')[0]
-  tinymce.activeEditor.selection.select(tinymce.activeEditor.dom.select('span#autocomplete')[0])
-  tinymce.activeEditor.execCommand('mceInsertContent', false, `${item.mention}&nbsp;`)
-  tinymce.activeEditor.dom.remove(selection)
+  if (!item.mention) {
+    console.log('BUG: this member does not have a username')
```

Comment:
Is this called when a user without username is clicked on? Seems strange to me to silenc this, but if I remember right it has been discussed, isn't it?

Reply:
This should not happen, because in this PR, I also hide users that have no username. But anyway, I wanted to extra check this "just in case"

Reply:
OK